pkg/load/load.go

```diff
@@ -232,3 +233,14 @@ func loadWorkflow(bytes []byte) (string, string, api.MultiStageTestConfiguration
 	}
 	return workflow.Workflow.As, workflow.Workflow.Documentation, workflow.Workflow.Steps, nil
 }
+
+func ClusterProfilesConfig(configPath string, profiles *api.ClusterProfilesList) error {
```

Comment:
Let's merge the existing profiles from `pkg/api` with the loaded configuration here so that we just need to change this in the future.

Reply:
Also, as much as I like the homage to C, the usual way to do this is to return multiple values.  So this function should receive just the path and return `(api.ClusterProfileList, error)`.